README.md

```diff
@@ -582,7 +582,7 @@ let layout = { (view1: UIView, view2: UIView) in
 
 ### Void arguments/return types
 
-It is preferable to not use `Void` in closure arguments or return types, omitting parentheses where possible.
+It is preferable to omit `Void` in closure return types.
```

Comment:
This line should still include `closure arguments` e.g. in the example `noArgClosure`